xep-0424.xml

```diff
@@ -20,13 +20,18 @@
     <spec>XMPP IM</spec>
     <spec>XEP-0313</spec>
     <spec>XEP-0421</spec>
-    <spec>XEP-0422</spec>
   </dependencies>
   <supersedes/>
   <supersededby/>
   <shortname>message-retract</shortname>
   &lance;
   &jcbrand;
+  <revision>
+    <version>1.0.0</version>
```

Comment:
Hmm. Next version is 0.4.0. 1.0.0 will be used once the XEP moves to stable.

Reply:
Thanks, fixed.